exotica/src/Scene.cpp

```diff
@@ -689,7 +689,7 @@ void Scene::addObject(const std::string& name, const KDL::Frame& transform, cons
     if (updateCollisionScene) updateCollisionObjects();
 }
 
-void Scene::addObjectToEnvironment(const std::string& name, const KDL::Frame& transform, shapes::ShapeConstPtr shape, const bool updateCollisionScene)
+void Scene::addObjectToEnvironment(const std::string& name, const KDL::Frame& transform, shapes::ShapeConstPtr shape, const std_msgs::ColorRGBA& colour, const bool updateCollisionScene)
```

Comment:
Ok. I assume that the python binding for `std_msgs::ColorRGBA` can be removed too?

Reply:
Yes, absolutely

Reply:
I just saw you used `Eigen::Vector4f colour = Eigen::Vector4d()`. Please use double consistently

Reply:
`std_msgs::ColorRGBA` uses floats. I would like to use float vectors to prevent all the compiler warnings related to the precision loss for double to float conversions.

Reply:
We've got everything in double already so, yes, while we technically do not need it let's stay consistent for now.